standards_lab/ui/static/theme.css

```diff
@@ -0,0 +1,7838 @@
+/*! `Custom` Bootstrap 4 theme */
```

Comment:
Could we add the info from this commit message https://github.com/OpenDataServices/standards-lab/pull/28/commits/65dbd817a522d1d8e04a6db4a4b13e99f71d86e2 at the top of this file?